### PR TITLE
Model Viewer 1.0 blog post initial draft

### LINKED ISSUE #3322: content: Web 3D Made Simple with <model-viewer> 1.0 [2020-06-26]
* [draft](https://docs.google.com/document/d/1RJ6xge9RV_SAO5CtXJJNYkxvlqeSBhFDwIi-UBku6X4/edit?ts=5ee3be96#)
* [thread](https://groups.google.com/a/google.com/g/webdev-content/c/N3vpC9hEJeI)

### PR DESCRIPTION
**Pull request by rkochman**
Fixes #3322

### CODE REVIEW

File: src/site/content/en/blog/model-viewer-10/index.md [hunk 1]
```diff
@@ -1,18 +1,94 @@
...
+</model-viewer>
+```
+
+<iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://modelviewer.dev/webdotdev/annotations.html" frameborder="0" allowfullscreen></iframe>
+
+See the [annotations documentation page](https://modelviewer.dev/examples/annotations.html) for more information.
+
+### Tester ###
+
+We also created and host a `<model-viewer>` [“tester” tool](https://modelviewer.dev/examples/tester.html), which enables you to quickly preview your model, try out different `<model-viewer>` configurations (e.g. exposure and shadow softness), generate a poster image, and interactively get coordinates for annotations.
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply src/site/content/en/blog/model-viewer-10/index.md:75]: We/our aren't allowed on web.dev. Please fix this throughout.

File: src/site/content/en/blog/model-viewer-10/index.md [hunk 2]
```diff
@@ -1,18 +1,94 @@
...
+<model-viewer camera-controls skybox-image="spruit_sunrise_1k_HDR.hdr" alt="A 3D model of a damaged helmet" src="DamagedHelmet.glb"></model-viewer>
+```
+
+<iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://modelviewer.dev/webdotdev/rendering.html" frameborder="0" allowfullscreen></iframe>
+
+### Stability ###
+
+With `<model-viewer>` reaching its first major version, we’re prioritizing API stability and will avoid making any breaking changes until version 2.0 is released. 
+
+## What’s next ##
```
Comment (jpmedley) [inline src/site/content/en/blog/model-viewer-10/index.md:91]:
```suggestion
## What’s next?
```

**Comment by jpmedley** [comment]
@kaycebasques This and the other model-viewer article are going to sit a while. Rob is out of the office until late in the month.

File: src/site/content/en/blog/model-viewer-10/index.md [hunk 3]
```diff
@@ -1,18 +1,94 @@
...
-hero: hero.jpg
-alt: A 3D model of a space suit.
 ---
 
-TBD
+3D models are more relevant than ever. Retailers are using 3D models to bring the in-store shopping experience to customers at home. Museums are making 3D models of their artifacts available to everyone on the web. Unfortunately, it can be difficult to add a 3D model to a website in a way that provides a great user experience, without a deep knowledge of 3D technologies or resorting to hosting 3D content on a third-party site. To address this issue, [early last year we introduced](https://developers.google.com/web/updates/2019/02/model-viewer) `<model-viewer>`, which seeks to make putting 3D models on the web as easy as writing a few lines of HTML. Since then, we’ve been working to address feedback and requests from the community. The culmination of that work is `<model-viewer>` version 1.0. 
 
+## What’s new since last year? ##
 
+Version 1.0 includes the most-requested features, improvements to speed and fidelity, and built-in support for augmented reality (AR) on the web.
```
[the remark this answers is not in the document]
Comment (rkochman) [reply src/site/content/en/blog/model-viewer-10/index.md:20]:
```suggestion
Version 1.0 includes built-in support for augmented reality (AR) on the web, improvements to speed and fidelity, and other frequently-requested features.
```

File: src/site/content/en/blog/model-viewer-10/index.md [hunk 4]
```diff
@@ -1,18 +1,94 @@
...
+
+### Augmented reality
+
+Viewing a 3D model on a blank canvas is great, but being able to view it in your space is even better. Now `<model-viewer>` supports WebXR augmented reality, for an entirely-within-the-browser 3D and AR experience on Chrome Android. 
+
+```html
+<model-viewer src="ToyTrain.glb" ar ar-scale="auto" camera-controls alt="A 3D model of a wooden toy train"> 
+</model-viewer>
+```
+<iframe style="width:100%; height: 100%;position: absolute; top: 50%; left: 50%; transform: translate(-50%,-50%);" src="https://modelviewer.dev/webdotdev/webxr.html" frameborder="0" allowfullscreen></iframe>
```
Comment (kaycebasques) [inline src/site/content/en/blog/model-viewer-10/index.md:30]: (Noting for myself) I wasn't sure if @robdodson would be OK with iframe-ing stuff into our content but from the looks of this content it seems OK: https://github.com/GoogleChrome/web.dev/pull/3289/files#r445240150

Comment (robdodson) [reply src/site/content/en/blog/model-viewer-10/index.md:30]: Yeah if folks need to demo something then iframes are the way to go. That's how we pull our glitch examples into the site.

You'll want to change the inline styles on all of these iframes as they attempt to make it fill the entire viewport. I'd suggest something like this:

```html
  <iframe
    src="https://modelviewer.dev/webdotdev/webxr.html"
    title="A toy train using model-viewer"
    style="height: 420px; width: 100%; border: 0;">
  </iframe>
```

**Pull request closed without merging**